### DISCUSSION TITLE
Detectron2 loss_box_reg = 0 and prediction returning empty tensor[]

### DISCUSSION THREAD

**Discussion by deeplearningnoobie** [facebookresearch/detectron2#3843] [Q&A]
While Training a custom dataset on _COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml_ I get the following output:

_iter: 219 total_loss: 0.04751 loss_cls: 0.000811 **loss_box_reg: 0** loss_rpn_cls: 0.03579 loss_rpn_loc: 0.007368 time: 0.9141 data_time: 0.0076 lr: 0.00024886 max_mem: 1832M_.

The loss_box_reg: 0 and while predicting the bbox:

_tensor([], device='cuda:0', dtype=torch.int64) Boxes(tensor([], device='cuda:0', size=(0, 4)_.

Any suggestions on the problem?

**Comment by sukjunhwang** [comment]
Maybe the custom dataset is not correctly registered, resulting in no ground-truth bounding box annotation fed to the detector.

**Comment by ArpanGyawali** [comment]
https://github.com/facebookresearch/detectron2/discussions/5354#discussion-7087539

@sukjunhwang can you help me with similar problem but loss_reg_loss is decreasing in my case posted in the discussion above. help me
